app/notifications/process_notifications.py

```diff
@@ -283,7 +283,7 @@ def send_notification_to_queue(notification, research_mode, queue=None):
         raise
 
     current_app.logger.info(
-        "{} {} sent to the {} queue for delivery".format(notification.notification_type, notification.id, queue)
+        "------ ---- {} {} sent to the {} queue for delivery".format(notification.notification_type, notification.id, queue)
```

Comment:
why did we need this? is this for checking the logs when this goes live?

Reply:
yeah, this log is just to ensure that we can 💯 see how a message was sent, but we definitely don't need all the dashes, that was just to help me see the lines locally. 😅  I shall get rid of them.